### ISSUE TITLE
After fine-tuning, the videos generated during the experiment exhibited a significant grid pattern.

### ISSUE THREAD

**Issue by Junsheng121** [THUDM/CogVideo#590]
![image](https://github.com/user-attachments/assets/951e4ca5-80a4-490f-b9ab-d1a2a383111e)
hello! I tried fine-tuning on CogvideoX 5B t2v model, after 300 epochs fine-tuning on my dataset consisting of 60 videos, it generates the grid pattern above. I wonder why this happens or whether I have missed something.
Thanks for your great contribution!

**Comment by gulucaptain** [comment]
Maybe you can try to reduce the learning rate or increase the training data and batch size.

**Comment by Junsheng121** [comment]
> Maybe you can try to reduce the learning rate or increase the training data and batch size.

Thanks. It works and the grid pattern disappears. BTW why the grid exists? Is it overfitting so that deviate from 3DVAE distribution?

**Comment by SSUHan** [comment]
@Junsheng121 
Which one worked for in this problem? Reducing lr? or increasing batch size? 
I'm also suffering from the same promblem now :(
I used batch size of 2 per GPU * 4 GPU so that total batch size is 8. And lr == 1e-4 or 5e-5

**Comment by Junsheng121** [comment]
> @Junsheng121 Which one worked for in this problem? Reducing lr? or increasing batch size? I'm also suffering from the same promblem now :( I used batch size of 2 per GPU * 4 GPU so that total batch size is 8. And lr == 1e-4 or 5e-5

just reducing training steps. I consider it an overfitting problem :)
